vllm-proxy/src/app/quote/quote.py

```diff
@@ -125,11 +123,14 @@ def build_payload(self, nonce, evidence, cert_chain):
         return payload
 
 
-quote = Quote(signing_method=SIGNING_METHOD)
-quote.init()
+ecdsa_quote = Quote(signing_method=ECDSA)
+ecdsa_quote.init()
+
+ed25519_quote = Quote(signing_method=ED25519)
+ed25519_quote.init()
```

Comment:
Note the quote has nothing related to the signer key type, so I would suggest have a `Signer` class handle the signing key generation and sign operation.

Can be refactored to something like below:

```
ecdsa_signer = Signer(signing_method=ECDSA)
ed25519_signer = Signer(signing_method=ED25519)

quote = Quote()
quote.init()
quote.set_ecdsa_signer(ecdsa_signer)
quote.set_ed25519_signer(ed25519_signer)
```